### PR TITLE
Ubuntu-20.04 image will be included in Pouta

### PR DESCRIPTION
**Pull request by khabiruddin**
We are going to provide Ubuntu-20.04 image for our Pouta customers.

CCCP-3301

### CODE REVIEW

File: docs/cloud/pouta/images.md [hunk 1]
```diff
@@ -88,6 +88,12 @@ do not want to use CentOS. Note that Ubuntu-18.04 is that one image that
 has `ubuntu` as the default username instead of `cloud-user`, since it was the
 first image type we started to provide directly from the upstream.
 
+### Ubuntu-20.04 LTS
+We will provide Ubuntu-20.04 LTS images which is the latest in Ubuntu image family for who
```
Comment (junousi) [inline docs/cloud/pouta/images.md:92]: Using future tense sounds weird, I would suggest trimming this to:

```
-We will provide Ubuntu-20.04 LTS images which is the latest in Ubuntu image family for who
+Ubuntu-20.04 LTS image is the latest in Ubuntu image family for those who
```

(also `those` added)

**Comment by junousi** [comment]
After this change, the following statement would be untrue:

`Note that Ubuntu-18.04 is that one image that has ubuntu as the default username`

**Review by junousi — CHANGES_REQUESTED** [review]
Couple of fixes plz

**Comment by khabiruddin** [comment]
> After this change, the following statement would be untrue:
> 
> `Note that Ubuntu-18.04 is that one image that has ubuntu as the default username`
> 
> So it needs changing too.

This is not my edition but I would like to change it.

File: docs/cloud/pouta/images.md (hunk 1 above)
Comment (khabiruddin) [reply docs/cloud/pouta/images.md]: There are some grammatical fixes.

**Comment by khabiruddin** [comment]
> Couple of fixes plz

Yes I have fixed those.

**Comment by khabiruddin** [comment]
> After this change, the following statement would be untrue:
> 
> `Note that Ubuntu-18.04 is that one image that has ubuntu as the default username`
> 
> So it needs changing too.

Yes done.

**Comment by junousi** [comment]
One more thing - can you amend Ubuntu 20.04 to the table on the same page, too

**Comment by khabiruddin** [comment]
> One more thing - can you amend Ubuntu 20.04 to the table on the same page, too

Yes done.

**Review by junousi — APPROVED** [review]
Looks good to me!

**Pull request merged**